unearthed-arcana/2016/20160912.xml

```diff
@@ -0,0 +1,759 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+    <info>
+        <name>Unearthed Arcana: The Ranger, Revised</name>
+        <description>A warrior who uses martial prowess and nature magic to combat threats on the edges of civilization..</description>
+        <author url="http://dnd.wizards.com/articles/features/unearthed-arcana-ranger-revised">Wizards of the Coast</author>
+        <update version="0.0.3">
+            <file name="20160912.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/unearthed-arcana/2016/20160912.xml" />
+        </update>
+    </info>
+    <element name="The Ranger, Revised" type="Class" source="Unearthed Arcana" id="ID_WOTC_UA_CLASS_The_Ranger,_Revised">
```

Comment:
You can leave the name of the class just as **Ranger** instead of Ranger Revised.

IDs should be all CAPS and not contain any special character other than the underscore. So the **,** shouldn't be included.